libsolidity/formal/SymbolicState.cpp

```diff
@@ -104,6 +104,14 @@ smtutil::Expression SymbolicState::blockhash(smtutil::Expression _blockNumber) c
 	return smtutil::Expression::select(m_tx.member("blockhash"), move(_blockNumber));
 }
 
+void SymbolicState::newBalances()
+{
+	auto stateS = dynamic_pointer_cast<TupleSort>(stateSort());
```

Comment:
For a moment I started wondering what this mysterious state `S` is :) How about this?
```suggestion
	auto tupleSort = dynamic_pointer_cast<TupleSort>(stateSort());
```

Reply:
Yea I wanted to call it `stateSort` but the function is already called like that. Will change